llvm/test/Assembler/x86mmx.ll

```diff
@@ -1,9 +0,0 @@
-; RUN: llvm-as < %s | llvm-dis | FileCheck %s
-; RUN: verify-uselistorder %s
-; Basic smoke test for x86_mmx type.
-
-; CHECK: define x86_mmx @sh16
-define x86_mmx  @sh16(x86_mmx %A) {
```

Comment:
I'd expect this to require movement to a bitcode auto upgrade test (but this also appears to be missing a call test)